packages/embark-console/src/lib/index.ts

```diff
@@ -1,14 +1,11 @@
-/*globals __*/
-import "colors";
-const env = require("../../core/env");
-const utils = require("../../utils/utils");
-const {escapeHtml, exit} = require("embark-utils");
-import { Callback } from "embark";
-import constants from "embark-core/constants.json";
-const stringify = require("json-stringify-safe");
 import { waterfall } from "async";
-import { Embark, Events } from "embark";
-import {__} from "i18n";
+import chalk from "chalk";
+import { Callback, Embark, Events } /* supplied by @types/embark in packages/embark-typings */ from "embark";
```

Comment:
This line doesn't actually import something from the `embark` package; rather, it's automagical behavior of `tsc` to get types from `@types/embark` which is in `packages/embark-typings`.

So, `embark` is not needed as a dependency (which would be circular) but `@types/embark` is needed as a devDep.

Reply:
I removed the dependency on `embark` when we decided to go against the global env handling for `CMD_HISTORY_SIZE`.

Reply:
It was still in the deps list, though maybe I had an old version of the commit, but I checked the sha and it seemed I was working on the last version you had pushed.